frontend/sw360-portlet/src/main/resources/META-INF/resources/js/modules/mergeWizard.js

```diff
@@ -120,28 +120,37 @@ define('modules/mergeWizard', [ 'jquery', 'modules/sw360Wizard' ], function($, s
             keys = [],
             existInBoth = [];
 
-        target = target == null ? {} : target;
-        source = source == null ? {} : source;
+        if (target == null) {
+            target = new Map();
+        } else {
+            target = target instanceof Map ? target : new Map(Object.entries(target));
+        }
+        if (source == null) {
+            source = new Map();
+        } else {
+            source = source instanceof Map ? source : new Map(Object.entries(source));
+        }
         detailFormatter = detailFormatter || function(element) { return element; };
 
         result = $($.parseHTML('<fieldset id="' + normalizePropName(propName) + '" class="merge block">' +
                                '    <h4>' + propName + '</h4>' +
                                '</fieldset>'));
 
-        $.each(target, function(key, value) {
-            if (!source[key]) {
+        target.forEach((value, key) => {
+            if(!source.get(key)) {
```

Comment:
Please check Indentation after if.

Reply:
@quan-tv ,We can add a space between the **if and (**    (please check the line 132) , its not an issue with the code but it is just the indentation we are maintaining.
like: if(!source.get(key)) {
 to   if (!source.get(key)) {

Reply:
Ah, I got it. I myself also prefer this coding convention. I will update it.

Reply:
@ravi110336 
The source code was updated!